src/shared/components/FileDrop/FileDrop.style.ts

```diff
@@ -0,0 +1,152 @@
+import { breakpoints, colors, sizes, transitions } from '@/shared/theme'
+import styled from '@emotion/styled'
+import Icon from '../Icon'
+import Text from '../Text'
+import { darken } from 'polished'
+import Button from '../Button'
+
+type DragAndDropAreaProps = {
+  isDragAccept?: boolean
+  isFileDialogActive?: boolean
+}
+
+type ProgressBarProps = {
+  progress?: number
+}
+
+export const DragAndDropArea = styled.div<DragAndDropAreaProps>`
+  position: relative;
+
+  background-color: ${darken(0.16, colors.gray[600])};
+  cursor: pointer;
+  width: 320px;
```

Comment:
I believe percentage value would be better for small devices. Or does it have to be fixed?